### ISSUE TITLE
migrations timestamp bug

### ISSUE THREAD

**Issue by Spoki4** [typeorm/typeorm#360]
Hi, u use in migrations table only 10 numbers. 

timestamp   | real_timestamp
--------------| -----------------
7111386161 | 1487111386161
7497664672 | 1487497664672
7802041914 | 1487802041914
7932192510 | 1487932192510
7943972051 | 1487943972051
9413161823 | 1489413161823
9519798663 | 1489519798663
9550577803 | 1489550577803

And now I try add new migrations with timestamp 1490149967376. 1490149967376 > 1489550577803 BUT
0149967376 < 9550577803. I got an error:
```
Error: New migration found: CreateOfferTable1490149967376, however this migration's timestamp is not valid. Migration's timestamp should not be older then migrations already executed in the database.
    at D:\Repositories\palport\src\migration\MigrationExecutor.ts:63:23
    at Array.filter (native)
    at MigrationExecutor.<anonymous> (D:\Repositories\palport\src\migration\MigrationExecutor.ts:55:49)
    at step (D:\Repositories\palport\node_modules\typeorm\migration\MigrationExecutor.js:32:23)
    at Object.next (D:\Repositories\palport\node_modules\typeorm\migration\MigrationExecutor.js:13:53)
    at fulfilled (D:\Repositories\palport\node_modules\typeorm\migration\MigrationExecutor.js:4:58)
    at process._tickDomainCallback (internal/process/next_tick.js:129:7)
```

**Comment by pleerock** [comment]
As I understood migrations are being generated properly, however when they are executed the timestamp saved in the database table is wrong. Am I right?

**Comment by Spoki4** [comment]
yes, in database migration timestamp cuts the first 3 digits.

**Comment by pleerock** [comment]
fixed and released in `0.0.10`. Please check that.

**Comment by valVk** [comment]
https://github.com/typeorm/typeorm-typedi-extensions/issues/9
